case-lib/lib.sh

```diff
@@ -67,13 +68,15 @@ func_lib_disable_pulseaudio()
     IFS="$OLD_IFS"
     [[ "${#PULSECMD_LST[@]}" -eq 0 ]] && return
     func_lib_check_sudo
-    [[ ! -f $PULSEAUDIO_CONFIG.bak ]] && sudo cp $PULSEAUDIO_CONFIG $PULSEAUDIO_CONFIG.bak -f
-    # because hijack the sudo command
-    if [ "$(grep 'autospawn' $PULSEAUDIO_CONFIG)" ]; then
-        sudo "sed -i '/autospawn/cautospawn = no' $PULSEAUDIO_CONFIG"
-    else
-        sudo "bash -c 'echo \"autospawn = no\" >> $PULSEAUDIO_CONFIG'"
-    fi
+    # get all running pulseaudio paths
+    PULSE_PATHS=( $(ps -C pulseaudio -o cmd --no-header | awk '{print $1}') )
+    for PA_PATH in $PULSE_PATHS
```

Comment:
`$PULSE_PATHS` is array, so you need `${PULSE_PATHS[@]}` to go though it

Reply:
thanks, done